### PR TITLE
feat(sg-sidebar): add sidebar details for sg groups

### PR DESCRIPTION
**Pull request by ksendart**
#577

### CODE REVIEW

**Comment by zolotyx** [comment]
@ksendart , templates cannot be used in VM's, so there is not need to show "VMs using the group" section

**Comment by zolotyx** [comment]
@ksendart , proposing to remove route for rules. let's just open modal? what do you think?

File: src/app/reducers/vm/redux/vm.reducers.ts [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+  selectAll,
+  selectTotal,
+} = adapter.getSelectors(getVMsEntitiesState);
+
+export const isLoading = createSelector(
+  getVMsEntitiesState,
+  state => state.loading
+);
+
+export const getUsingSGVolume = createSelector(
```
Comment (zolotyx) [inline src/app/reducers/vm/redux/vm.reducers.ts]: **SGVolume**, what is it?

File: src/app/security-group/sg-creation/security-group-creation.component.html [hunk 2]
```diff
@@ -1,3 +1,4 @@
+{{creationInProgress}}
```
Comment (zolotyx) [inline src/app/security-group/sg-creation/security-group-creation.component.html]: is it a debug information?

File: src/app/security-group/sg-list-item/row-item/security-group-row-item.component.ts [hunk 3]
```diff
@@ -1,26 +1,23 @@
...
+import { SecurityGroupListItemComponent } from '../security-group-list-item.component';
+import { MatMenuTrigger } from '@angular/material';
 
 @Component({
   selector: 'cs-security-group-row-item',
   templateUrl: 'security-group-row-item.component.html',
   styleUrls: ['security-group-row-item.component.scss']
 })
-export class SecurityGroupRowItemComponent implements OnChanges {
+export class SecurityGroupRowItemComponent extends SecurityGroupListItemComponent{
```
Comment (zolotyx) [inline src/app/security-group/sg-list-item/row-item/security-group-row-item.component.ts]: add missing whitespace

File: src/app/shared/components/grouped-list/grouped-list.component.scss [hunk 4]
```diff
@@ -14,6 +14,10 @@
     font-size: 24px;
     margin: 0;
     padding-bottom: 15px;
+
+    &.no-padding {
```
Comment (zolotyx) [inline src/app/shared/components/grouped-list/grouped-list.component.scss]: why is it required?

File: src/app/shared/components/grouped-list/grouped-list.component.html [hunk 5]
```diff
@@ -1,11 +1,12 @@
 <ng-container *ngFor="let child of tree">
   <div class="grouped-section" *ngIf="child.name; else leaf">
-    <h4>{{ child.name | translate }}</h4>
+    <h4 [ngClass]="{ 'no-padding': isTable }">{{ child.name | translate }}</h4>
     <cs-grouped-list
       [component]="component"
       [groupings]="groupings"
       [level]="level + 1"
       [list]="child.items"
+      [isTable]="isTable"
```
Comment (zolotyx) [inline src/app/shared/components/grouped-list/grouped-list.component.html]: can we get rid of this?

File: src/app/reducers/vm/redux/vm.reducers.ts (hunk 1 above)
Comment (ksendart) [reply src/app/reducers/vm/redux/vm.reducers.ts]: oops...

File: src/app/security-group/sg-list-item/card-item/security-group-card-item.component.scss [hunk 6]
```diff
@@ -1,5 +1,6 @@
 :host {
   display: inline-flex;
+  cursor: pointer;
```
Comment (zolotyx) [inline src/app/security-group/sg-list-item/card-item/security-group-card-item.component.scss]: please add this to mat-card child

File: src/app/security-group/sg-list-item/row-item/security-group-row-item.component.scss [hunk 7]
```diff
@@ -1,3 +1,7 @@
 .desc {
   flex-grow: 2;
 }
+
+:host {
+  cursor: pointer;
```
Comment (zolotyx) [inline src/app/security-group/sg-list-item/row-item/security-group-row-item.component.scss]: maybe add this to a child?

**Pull request merged**